templates/Image classification_PyTorch/code-template.py.jinja

```diff
@@ -6,6 +6,7 @@
 #
```

Comment:
@jrieke I don't quite understand why the `pip install` command below is not placed here when using a notebook instance (e.g in Jupyter `!pip install ...`).